guides/doc-Project_API/topics/proc_configuring-ssl-authentication.adoc

```diff
@@ -0,0 +1,35 @@
+[id="configuring-ssl-authentication"]
+= Configuring SSL authentication
+
+Configure an SSL authentication for the API requests to {ProjectServer}.
+
+.Procedure
+.  Obtain a certificate from the {ProjectServer} with which you want to communicate using one of the following options:
+* If you execute the command from a remote server, obtain a certificate using SSH:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ scp root@_{foreman-example-com}_:/var/www/html/pub/katello-server-ca.crt /etc/pki/ca-trust/source/anchors/_{foreman-example-com}_-katello-server-ca.crt
+----
+* If you execute the command directly on the {ProjectServer}, copy the certificate to the `/etc/pki/ca-trust/source/anchors` directory:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ cp /var/www/html/pub/katello-server-ca.crt /etc/pki/ca-trust/source/anchors/_{foreman-example-com}_-katello-server-ca.crt
+----
+.  Add the certificate to the list of trusted CAs:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+update-ca-trust extract
+----
+
+.Verification
+* Verify that the certificate is present in the NSS database by entering the API request without the `--cacert` option:
```

Comment:
what does "NSS database" mean?

maybe reword to "Verify that your client trusts the {Project} SSL certificate"? <- if correct; someone please ACK :)

Reply:
I think you're right. I've found "NSS database" within [DS docs](https://docs.redhat.com/en/documentation/red_hat_directory_server/11/html/administration_guide/managing_the_nss_database_used_by_directory_server#managing_the_nss_database_used_by_directory_server)